### DISCUSSION TITLE
How to change paper orientation in print dialog?

### DISCUSSION THREAD

**Discussion by bubuta-bokh** [tannerhelland/PhotoDemon#542] [Q&A] [answered]
Can't find a way to change default landscape orientation to portrait one before printing.

**Comment by tannerhelland** [comment, marked as answer]
Hello @bubuta-bokh .  What version of Windows are you using?

On most versions of Windows, PhotoDemon just sends images to the built-in Windows Photo Printer tool when printing.  The features available depend on the version of Windows you're using (and possibly the attached printer).  Windows always tries to fit the image as best it can.  It's frustrating that Microsoft does not provide more control over this.

As a workaround, you may be able to adjust first the image size (`Image > Resize`) then the canvas size in PhotoDemon (`Image > Canvas Size`) to achieve your goal.  For example, if you want to print a 8.5" x 11" document as an 11" x 8.5" one, you can first resize the image (with aspect ratio preserved - the little lock icon set) to 8.5" tall.  PhotoDemon will automatically calculate the other dimension as 6.57".  Hit OK, then go to `Image > Canvas size` and set the canvas to 11" wide (with aspect ratio NOT preserved - the little lock icon set to unlocked).

Now the image should print in the opposite orientation.

This is a hacky workaround and I don't like it, but it may be better than nothing.
